test/integration/model.test.js

```diff
@@ -1436,7 +1436,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       await this.sequelize.sync({ force: true });
       await User.bulkCreate([{ username: 'user1' }, { username: 'user2' }]);
       const affectedRows = await User.destroy({ where: {} });
-      expect(affectedRows).to.equal(2);
+      expect(affectedRows).to.be.a('number').to.equal(2);
```

Comment:
There is no need to check that the value is a number if you already check that is equals a number value

```suggestion
      expect(affectedRows).to.equal(2);
```

Same comment for the rest of the changes in this file